networking.adoc

```diff
@@ -0,0 +1,1081 @@
+----
+
+## Updating node network configuration
+
+[source,terminal]
+----
+$ oc apply -f <br1-eth1-policy.yaml> 
+----
+
+
+[source,terminal]
+----
+$ oc apply -f <br1-eth1-policy.yaml> 
+----
+
+## Load balancing on RHOSP
+
+[source,terminal]
+----
+$ oc -n openshift-kuryr edit cm kuryr-config
+----
+
+
+[source,terminal]
+----
+$ oc -n openshift-kuryr edit cm kuryr-config
+----
+
+
+[source,terminal]
+----
+$ oc apply -f external_router.yaml
+----
+
+## Assigning a secondary network to a VRF
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+## Attaching a pod to an additional network
+
+[source,terminal]
+----
+$ oc create -f <name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc edit pod <name>
+----
+
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+
+[source,terminal]
+----
+$ oc edit pod <name>
+----
+
+## Configuring a bridge network
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+## Configuring a host-device network
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+## Configuring an ipvlan network
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+## Configuring a macvlan network with basic customizations
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+## Configuring a macvlan network
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+## Editing an additional network
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+## Removing an additional network
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+## Removing a pod from an additional network
+
+[source,terminal]
+----
+$ oc delete pod <name> -n <namespace>
+----
+
+## Understanding multiple networks
+
+[source,terminal]
+----
+$ oc delete pod <name> -n <namespace>
+----
+
+## Creating a network policy
+
+[source,terminal]
+----
+$ oc create -f <policy-name>.yaml -n <project>
+----
+
+## Creating default network policies for a new project
+
+[source,terminal]
+----
+$ oc adm create-bootstrap-project-template -o yaml > template.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f template.yaml -n openshift-config
+----
+
+
+[source,terminal]
+----
+$ oc edit template <project_template> -n openshift-config
+----
+
+
+[source,terminal]
+----
+$ oc new-project <project> 
+----
+
+## Deleting a network policy
+
+[source,terminal]
+----
+$ oc delete networkpolicy <policy-name>
+----
+
+## Editing a network policy
+
+[source,terminal]
+----
+$ oc apply -f <policy-file>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc edit <policy-name> -n <namespace>
+----
+
+## Configuring egress IPs for a project
+
+[source,terminal]
+----
+ $ oc patch netnamespace <project_name> --type=merge -p \ 
+  '{
+    "egressIPs": [
+      "<ip_address>" 
+    ]
+  }'
+----
```

Comment:
Requires declarative alternative

Reply:
For this item and the others related to Netnamespace and Hostsubnet, I was unable to apply a manifest equivalent.  I bugged forum-sdn and Dan Winship indicated the following:

> I think it’s because NetNamespace is a CRD type rather than a built-in type. You need to explicitly specify to use JSON rather than protobuf on the command line.

I don't plan to investigate any further for Netnamespace and Hostsubnet commands.